### PR TITLE
direct 'Get support' link to support.tidepool.org instead of oft-broken mailto link

### PR DESCRIPTION
**Pull request by jebeck**
What it says on the tin. I believe the consensus was just to get quick approval on this and merge to master right away and let it go out with the next release, remembering to add a small note to test the new functionality?

CC @darinkrauss for interpretation of process and quick approval?

### CODE REVIEW

**Comment by jebeck** [comment]
[Update "Get Support" link on Uploader to go to support.tidepool.org](https://trello.com/c/ZCjlxFzb/128-update-get-support-link-on-uploader-to-go-to-support-tidepool-org)

File: lib/pages/App.js [hunk 1]
```diff
@@ -277,7 +277,7 @@ export class App extends Component {
       <div className={styles.footerRow}>
         <div className={styles.version}>{`v${version} beta`}</div>
         <div className="mailto">
-          <a className={styles.footerLink} href="mailto:support@tidepool.org?Subject=Feedback on Uploader" target="mailto">Get support</a>
+          <a className={styles.footerLink} href="http://support.tidepool.org/" target="_blank">Get support</a>
```
Comment (darinkrauss) [inline lib/pages/App.js]: @jebeck Will there be a secure -> insecure warning from this? Of course, https://support.tidepool.org also gives an certificate warning, so I hope not.

Comment (jebeck) [inline lib/pages/App.js]: I didn't get any warnings when I tried it. It's from the uploader, which is an app, not an https site.

**Comment by jebeck** [comment]
I'll let you merge @gniezen since you've sequencing builds of the uploader for testing. But LMK if you'd rather I did/had done it!

File: lib/pages/App.js (hunk 1 above)
Comment (darinkrauss) [inline lib/pages/App.js]: Ah, right! LGTM!

**Comment by darinkrauss** [comment]
:shipit:

**Pull request merged**